go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/alicebob/miniredis/v2 v2.23.0
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/defsec v0.82.6
+	github.com/aquasecurity/defsec v0.82.7-0.20221225070347-3a6cfb67e460
```

Comment:
@chen-keinan - this PR introduced a version of defsec that isn't released, I'm curious what version is this? The released are here https://github.com/aquasecurity/defsec/releases